vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/WindowSteps.java

```diff
@@ -16,25 +16,42 @@
 
 package org.vividus.steps.ui.web;
 
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.not;
+
+import java.util.function.BiConsumer;
+
 import org.jbehave.core.annotations.When;
 import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.vividus.monitor.TakeScreenshotOnFailure;
 import org.vividus.selenium.IWebDriverProvider;
 import org.vividus.selenium.manager.IWebDriverManager;
 import org.vividus.softassert.ISoftAssert;
+import org.vividus.ui.web.action.IAlertActions;
+import org.vividus.ui.web.action.WebJavascriptActions;
 
 @TakeScreenshotOnFailure
 public class WindowSteps
 {
+    private static final Logger LOGGER = LoggerFactory.getLogger(WindowSteps.class);
+
     private final IWebDriverProvider webDriverProvider;
     private final IWebDriverManager webDriverManager;
     private final ISoftAssert softAssert;
+    private final WebJavascriptActions javascriptActions;
+    private final IAlertActions alertActions;
 
-    public WindowSteps(IWebDriverProvider webDriverProvider, IWebDriverManager webDriverManager, ISoftAssert softAssert)
+    public WindowSteps(IWebDriverProvider webDriverProvider, IWebDriverManager webDriverManager, ISoftAssert softAssert,
+            WebJavascriptActions javascriptActions, IAlertActions alertActions)
```

Comment:
```suggestion
    public WindowSteps(IWebDriverProvider webDriverProvider, IWebDriverManager webDriverManager, WebJavascriptActions javascriptActions, IAlertActions alertActions, ISoftAssert softAssert)
```

Reply:
the same order should be applied to fields and the order of fields initialization in the  conctructor